src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/CosmosContainerProvider.cs

```diff
@@ -37,6 +40,9 @@ public class CosmosContainerProvider : IHostedService, IRequireInitializationOnF
             CosmosDataStoreConfiguration cosmosDataStoreConfiguration,
             IOptionsMonitor<CosmosCollectionConfiguration> collectionConfiguration,
             ICosmosClientInitializer cosmosClientInitializer,
+            ICollectionSetup collectionSetup,
+            ICollectionDataUpdater collectionDataUpdater,
```

Comment:
Add validations to make sure these parameters are not null.

Reply:
done